main.js

```diff
@@ -7,6 +7,33 @@ const title = document.querySelector('#title');
 const author = document.querySelector('#author');
 const btn = document.querySelector('.btn');
 const bookHolder = document.querySelector('#bookContainer');
+const list = document.querySelector('#list');
+const add = document.querySelector('#add');
+const contact = document.querySelector('#contact');
+
+const dt = new Date();
```

Comment:
- [ ] You are supposed to implement the date logic using the luxon library as from the requirements. [luxon](https://moment.github.io/luxon/docs/manual/install.html)

Reply:
This issue is well resolved! ✔️